pkg/k8scfgmap/watch.go

```diff
@@ -0,0 +1,87 @@
+package k8scfgmap
```

Comment:
How about keeping it with other watcher `k8scfgwatch` and rename to `k8swatch`

Reply:
Moved it to the `k8scfgwatch` package.